packages/form-layout/src/vaadin-form-layout-mixin.js

```diff
@@ -385,6 +390,29 @@ export const FormLayoutMixin = (superClass) =>
         });
     }
 
+    /** @private */
+    __updateCSSGridLayout() {
+      let resetColumn = false;
+
+      [...this.children]
+        .flatMap((child) => {
+          return child.localName === 'vaadin-form-row' ? [...child.children] : [child];
+        })
+        .forEach((child) => {
+          if (child.localName === 'br') {
+            resetColumn = true;
+            return;
+          }
```

Comment:
Let's say someone adds a form-layout with a configuration like the following:

```html
<vaadin-form-layout auto-responsive auto-rows max-columns="2">
  <vaadin-form-row>
    <vaadin-text-field label="Username"></vaadin-text-field>
  </vaadin-form-row>
  <vaadin-text-field label="Email"></vaadin-text-field>
</vaadin-form-layout>
```

How would you expect form-layout to behave?
1. The "Email" field is placed next to the "Username" field on the same row; or
2. The "Email" field is placed on the next row, in the first column

With the proposed changes, option (1) would happen, but I wonder if it placing a `<vaadin-form-row>` should force a "line break", just like `<br>` elements do. WDYT?

Reply:
Good point, I agree. Updated the implementation to force a line break after `<vaadin-form-row>` elements